src/window.rs

```diff
@@ -206,6 +236,12 @@ impl WindowWrapper {
     pub fn handle_keyboard_input(&mut self, keycode: Option<Keycode>, text: Option<String>) {
         let modifiers = self.context.keyboard().mod_state();
 
+        if let Some(key) = keycode {
+            if key == Keycode::Return && modifiers.contains(Mod::LALTMOD) {
```

Comment:
This isn't very vim friendly. This should ideally be implemented as a setting or function inside of vim so that users can create their own key bindings rather than mandating that AltEnter has this function. Alt Enter can be bound inside of vim which would be overridden by this change.

Reply:
I agree it probably shouldn't be hardcoded, I'm not overly familiar with the codebase, but I'll investigate and see if I can make it work as a setting like the other init.vim settings